packages/fluentui/react-northstar/src/components/SplitButton/SplitButtonToggle.tsx

```diff
@@ -0,0 +1,174 @@
+import { Accessibility, buttonBehavior } from '@fluentui/accessibility';
+import * as customPropTypes from '@fluentui/react-proptypes';
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import {
+  childrenExist,
+  createShorthandFactory,
+  UIComponentProps,
+  ChildrenComponentProps,
+  commonPropTypes,
+  rtlTextContainer,
+  ContentComponentProps,
+} from '../../utils';
+
+import {
+  ComponentEventHandler,
+  WithAsProp,
+  withSafeTypeForAs,
+  FluentComponentStaticProps,
+  ProviderContextPrepared,
+} from '../../types';
+import { getElementType, useAccessibility, useStyles, useTelemetry, useUnhandledProps } from '@fluentui/react-bindings';
+// @ts-ignore
+import { ThemeContext } from 'react-fela';
+
+export interface SplitButtonToggleProps extends UIComponentProps, ContentComponentProps, ChildrenComponentProps {
+  /** Accessibility behavior if overridden by the user. */
+  accessibility?: Accessibility;
+
+  /** A button can show that it cannot be interacted with. */
+  disabled?: boolean;
+
+  /**
+   * Called after a user clicks the button.
+   * @param event - React's original SyntheticEvent.
+   * @param data - All props.
+   */
+  onClick?: ComponentEventHandler<SplitButtonToggleProps>;
+
+  /**
+   * Called after a user focuses the button.
+   * @param event - React's original SyntheticEvent.
+   * @param data - All props.
+   */
+  onFocus?: ComponentEventHandler<SplitButtonToggleProps>;
+
+  /** A button can emphasize that it represents the primary action. */
+  primary?: boolean;
+
+  /** A button can emphasize that it represents an alternative action. */
+  secondary?: boolean;
+}
+
+export type SplitButtonToggleStylesProps = Pick<SplitButtonToggleProps, 'primary' | 'disabled'>;
+
+const SplitButtonToggle: React.FC<WithAsProp<SplitButtonToggleProps>> &
+  FluentComponentStaticProps<SplitButtonToggleProps> = props => {
+  const context: ProviderContextPrepared = React.useContext(ThemeContext);
+  const { setStart, setEnd } = useTelemetry(SplitButtonToggle.displayName, context.telemetry);
+  setStart();
+
+  const {
+    accessibility,
+    // @ts-ignore
+    active,
+    as,
+    children,
+    content,
+    disabled,
+    primary,
+    className,
+    styles,
+    variables,
+    design,
+  } = props;
+
+  const hasChildren = childrenExist(children);
+
+  const getA11Props = useAccessibility(accessibility, {
+    debugName: SplitButtonToggle.displayName,
+    mapPropsToBehavior: () => ({
+      as,
+      active,
```

Comment:
```suggestion
      active: undefined,
```

And remove usage of `active` in `props`

Reply:
It's not even used anymore in `buttonbehavior`. Will remove it.